README.md

```diff
@@ -1,8 +1,44 @@
 # Slack help bot
 
-Listens for new posts in a designated Slack channel and raises a coresponding request in Jira. 
+Listens for new posts in a designated Slack channel and raises a coresponding request in Jira.
 
-## Getting Started
+## Creating the Slack App 
+<details>
+  <summary>Steps</summary>
+
+1. Create a new app in your workspace.
+
+<img src="images/step1.png" width=50% height=50% />
+
+2. Head to socket mode and enable it. You will then be asked to create a new token (call it **jira-integration**) This will only have **connections:write** in the scope. Select **Generate**. Make sure to write down the generated token as this will be required for the slack-help-bot configuration.
```

Comment:
```suggestion
2. Head to socket mode and enable it. You will then be asked to create a new token (call it **jira-integration**) This will only have **connections:write** in the scope. Select **Generate**. Copy the generated token as this will be required for the slack-help-bot configuration.
```